### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by triptych** [CatR3kd/Multiplayer-Text-Adventure#1]
Can you add a license md file so folks can know if / how they might use or contribute?

**Comment by CatR3kd** [comment]
> Can you add a license md file so folks can know if / how they might use or contribute?

I'm not going to add a liscense yet, but I'll probably have an MD file on contributions when I release the first version.
Out of curiosity though, why?

**Comment by triptych** [comment]
You add a license so that folks  know  how they can use (  or not use ) your code. For instance if you don't care if folks remix your code you might add a MIT license. If you want to the code to stay your copyright and disallow anyone you might add a more restrictive license. You need this because your repo is public and it allows anyone to see your code. You want to be able to indicate to folks how they might use it.  https://snyk.io/learn/open-source-licenses/

**Comment by CatR3kd** [comment]
Yeah, I'm just leaving the license off until I'm sure what I want to use, since with no liscense all rights are reserved.
